src/update_xs_yum.ml

```diff
@@ -435,29 +438,25 @@ let run root branch source_iso s3bin =
     (fun (binpkg_iso, sources_iso) ->
       Printf.printf "Running createtree\n%!";
       create_tree root binpkg_iso sources_iso
-      >>|= fun () ->
+      >>= fun () ->
       Printf.printf "Running createrepo\n%!";
-      check (Lwt_unix.system (Printf.sprintf "createrepo %s/domain0" root))
-      >>|= fun _ ->
+      check_command (Printf.sprintf "createrepo %s/domain0" root)
+      >>= fun () ->
```

Comment:
what is the difference here of doing `fun () -> ...` vs `fun _ -> ...`?

Reply:
The first checks that the previous function returns a `unit` Lwt thread, so it's safer. With the second, it can happen that we accidentally ignore a return value for example.